### PR TITLE
Update conditional-access.md

### PR DESCRIPTION
**Pull request by ElaineAnsell**
Modified the recommended settings for CA and MFA policies to be more accurate and with more detailed guidance to reduce customer security risk with these changes

@jaimeo , I made some additional changes to this docs article after consulting with both CAT and our MMD security experts, Florent and Dan. Thanks for reviewing

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

**Comment by ElaineAnsell** [comment]
Adding Jaime as our product technical writer.  Apologies, I thought I had tagged Jaime in my PR to review the copy.

Thank you!

Regards,
Elaine

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by jaimeo** [comment]
Do not merge until I can edit.

**Comment by jaimeo** [comment]
@ElaineAnsell can you confirm my edits have not altered the technical accuracy? LMK either over Teams or by comment here.

Note that the audience we are writing to _is_ "admins in your company," therefore "you" is the right way to address them. Also a reminder that the product name is now "Microsoft Defender for Endpoint."

File: microsoft-365/managed-desktop/get-started/conditional-access.md [hunk 1]
```diff
@@ -27,9 +27,9 @@ After you’ve completed enrollment in Microsoft Managed Desktop, some managemen
...
+- Conditional Access policies: If you create any new conditional access policies related to Azure AD, Microsoft Intune, or Microsoft Defender for Endpoint after Microsoft Managed Desktop enrollment, exclude the **Modern Workplace Service Accounts** Azure AD group from them. For steps, see [Conditional Access: Users and groups](https://docs.microsoft.com/azure/active-directory/conditional-access/concept-conditional-access-users-groups). Microsoft Managed Desktop maintains separate conditional access policies to restrict access to these accounts. To review the Microsoft Managed Desktop conditional access policy (**Modern Workplace – Secure Workstation**), got to Microsoft Endpoint Manager and navigate to **Conditional Access** in **Endpoint Security**. Don't modify any Azure AD conditional access policies created by Microsoft Managed Desktop that have "Modern Workplace" in the name.
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/get-started/conditional-access.md]:
```suggestion
- Conditional Access policies: If you create any new conditional access policies related to Azure AD, Microsoft Intune, or Microsoft Defender for Endpoint after Microsoft Managed Desktop enrollment, exclude the **Modern Workplace Service Accounts** Azure AD group from them. For steps, see [Conditional Access: Users and groups](https://docs.microsoft.com/azure/active-directory/conditional-access/concept-conditional-access-users-groups). Microsoft Managed Desktop maintains separate conditional access policies to restrict access to these accounts. To review the Microsoft Managed Desktop conditional access policy (**Modern Workplace – Secure Workstation**), go to Microsoft Endpoint Manager and navigate to **Conditional Access** in **Endpoint Security**. Don't modify any Azure AD conditional access policies created by Microsoft Managed Desktop that have "Modern Workplace" in the name.
```

File: microsoft-365/managed-desktop/get-started/conditional-access.md [hunk 2]
```diff
@@ -27,9 +27,9 @@ After you’ve completed enrollment in Microsoft Managed Desktop, some managemen
...
+- Multifactor authentication: If you create any new multifactor authentication requirements in conditional access policies related to Azure AD, Intune, or Microsoft Defender for Endpoint after Microsoft Managed Desktop enrollment, exclude the **Modern Workplace Service Accounts** Azure AD group from them. For steps, see [Conditional Access: Users and groups](https://docs.microsoft.com/azure/active-directory/conditional-access/concept-conditional-access-users-groups). Microsoft Managed Desktop maintains separate conditional access policies to restrict access to members of this group. To review the Microsoft Managed Desktop conditional access policy (**Modern Workplace -**), got to Microsoft Endpoint Manager and navigate to **Conditional Access** in **Endpoint Security**.
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/get-started/conditional-access.md]:
```suggestion
- Multifactor authentication: If you create any new multifactor authentication requirements in conditional access policies related to Azure AD, Intune, or Microsoft Defender for Endpoint after Microsoft Managed Desktop enrollment, exclude the **Modern Workplace Service Accounts** Azure AD group from them. For steps, see [Conditional Access: Users and groups](https://docs.microsoft.com/azure/active-directory/conditional-access/concept-conditional-access-users-groups). Microsoft Managed Desktop maintains separate conditional access policies to restrict access to members of this group. To review the Microsoft Managed Desktop conditional access policy (**Modern Workplace -**), go to Microsoft Endpoint Manager and navigate to **Conditional Access** in **Endpoint Security**. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**